src/app/components/cds/mockups/DeviceMockupComponent.module.css

```diff
@@ -0,0 +1,82 @@
+.device {
+  @mixin typography-subtitle1;
+  border: solid 2px var(--color-white-100);
+  border-radius: var(--border-radius-max);
+  color: var(--color-gray-15);
+  height: 932px;
+
+  .phoneWidth {
```

Comment:
@vasconsaurus didn't get a chance to comment on this before merge, but for both the phone and tablet you need an `&.phoneWidth {` and `&.tabletWidth` to get the widths to change since you are chaining together classes to the class `.device`, as written `.phoneWidth` and `.tabletWidth` think they are being applied to children of `.device`, not chained to it

Reply:
Thank you! Me and Alex are pairing on this, and were wondering what was amiss 😅

Reply:
no problem!
...and to take it to the next level, in general for css cascading you would put these & statements at the bottom of the `.device` block.

That way you can override anything above. For example if you look at the designs, this little area at the bottom is not present in tablet mode. So if you put these statements at the bottom, you could use them to not just change the widths, but also  hide .device home when in tablet mode

<img width="1021" alt="Screenshot 2024-11-21 at 1 33 38 PM" src="https://github.com/user-attachments/assets/4f6f1855-883c-48f1-80e8-b04e1ac25e55">